lib/src/lints/avoid_final_with_getter/visitors/avoid_final_with_getter_visitor.dart

```diff
@@ -24,10 +24,22 @@ class AvoidFinalWithGetterVisitor extends RecursiveAstVisitor<void> {
       final visitor = GetterVariableVisitor(node);
       node.parent?.accept(visitor);
 
-      if (visitor.hasVariable) {
-        _getters.add(node);
+      if (visitor.variable != null) {
+        _getters.add(FinalWithGetterInfo(node, visitor.variable!));
```

Comment:
Please, extract `visitor.variable` into a separate variable to avoid bang operator.
```suggestion
      final variable = visitor.variable;
      if (variable != null) {
        _getters.add(FinalWithGetterInfo(node, visitor.variable));
```